### PR TITLE
Create dependencies.py

### PR DESCRIPTION
**Pull request by krishpranav**
Hi I have added a new python file 
this will help the user to install all the dependencies quickly

### CODE REVIEW

File: dependencies.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import os
+import time
+
+Y = set(['yes', 'y', 'YES', 'Y'])
+N = set(['no', 'n', 'NO', 'N'])
+
+os.system("apt install figlet")
+os.system("figlet INSTALLATION")
+os.system("clear")
```
Comment (Omicron166yt) [inline dependencies.py:11]: Why generate a banner if right after you are going to remove it?
For that it is better not to put any banner

File: dependencies.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    os.system("pip install lolcat")
+    os.system("clear")
+    os.system("figlet INSTALLATION COMPLETED...")
+    print("you can now run infect.sh")
+  elif agree in N:
+    print("This tool needs some depedencies please run python3 dependencies.py")
+    print("exiting")
+    exit()
+    
+agreement()
```
Comment (Omicron166yt) [inline dependencies.py:32]: Correct me if I'm wrong
It is not necessary to create a function in this case as it will only be used once

**Review by Omicron166yt — CHANGES_REQUESTED** [review]
There are 2 things that don't make a lot of sense but the code is good

**Comment by noob-hackers** [comment]
Thanks for suggesting but we used bash Scripting not Python so ...we dont
need python code 🙄

**Comment by krishpranav** [comment]
ok thank you

**Pull request closed without merging**